pkg/metrics/reporter.go

```diff
@@ -200,13 +200,13 @@ func (r *Reporter) ReportPreflightsBypassed(ctx context.Context, output prefligh
 		NodeName:        getHostname(),
 		PreflightOutput: string(outputJSON),
 	}
-	go Send(ctx, r.baseURL, ev)
+	Send(ctx, r.baseURL, ev)
 }
 
 // ReportSignalAborted reports that a process was terminated by a signal.
 func (r *Reporter) ReportSignalAborted(ctx context.Context, signal os.Signal) {
 	ev := r.newGenericEvent(types.EventTypeSignalAborted, signal.String(), true)
-	go Send(ctx, r.baseURL, ev)
+	Send(ctx, r.baseURL, ev)
```

Comment:
if this happens in the background it doesnt actually send because the process exits before it can